hax/hax/util.py

```diff
@@ -449,8 +455,10 @@ def get_leader_session_no_wait(self) -> str:
         Returns the RC leader session. HAConsistencyException is raised
         immediately if there is no RC leader selected at the moment.
         """
-        leader = self.kv.kv_get('leader')
         try:
+            leader = self.kv.kv_get('leader')
```

Comment:
why not use `allow_null=False` here as well?

Reply:
Because this function is already raising as exception if value in None